sdk/storage/azure-storage-blob/tests/test_common_blob_async.py

```diff
@@ -1142,6 +1143,7 @@ def test_soft_delete_with_leased_blob(self):
         # Assert
         self.assertIsNotNone(copy)
         self.assertEqual(copy['copy_status'], 'success')
+        self.assertFalse(isinstance(copy['copy_status'], Enum))
```

Comment:
I would like to assert it is a string more than just not enum. :)

Reply:
It's the first thing I did, but the original is a string AND an enum (multiple inheritance). So both test were True :)
Being that the line before is testing the string, I think we're good :)

Reply:
@xiangyan99 - that's already implied by the assertEqual statement above?